installer/docker/Dockerfile

```diff
@@ -83,17 +90,18 @@ LABEL org.label-schema.build-date=$BUILD_DATE \
       org.label-schema.vcs-ref=$VCS_REF \
       org.label-schema.schema-version=$VCS_TAG
 
-# install NEURON
-RUN sudo pip install NEURON
+# install hnn-core
+RUN pip install hnn-core
+
+RUN sudo apt-get update && \
+    sudo apt-get install -y neuron-dev
```

Comment:
Out of curiosity, is this preferred over doing a pip installation of NEURON?

Reply:
Just from experience I was having problems installing neuron with pip3, and so from some online resources (https://command-not-found.com/nrnivmodl) I tried an alternative and it worked

Reply:
Hmm, I've never run into issues using pip except on Windows (without WSL). NEURON doesn't currently distribute a python wheel for windows, but it should work for WSL, native linux, and macOS.

Reply:
This is the error I run into 

```
9.505 Processing triggers for libc-bin (2.31-0ubuntu9.12) ...
9.545 Cloning into '/home/hnn_user/hnn_source_code'...
10.82 nrnivmodl mod
10.82 make: nrnivmodl: Command not found
10.82 make: *** [Makefile:11: x86_64/special] Error 127

```

Reply:
To further clarify, I run into this error attempting to build the image on my mac

Reply:
I guess the "barebone" mac install in a docker is not the same as an actual mac computer ... there may be some compilers etc missing